core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -189,84 +182,8 @@ public S3RestServiceHandler(@Context ServletContext context)
-          e.getMessage(), S3ErrorCode.INTERNAL_ERROR.getStatus()));
-    }
-  }
-
-  /**
-   * Get username from parsed header info.
-   *
-   * @return username
-   * @throws S3Exception
-   */
-  private String getUserFromSignature() throws S3Exception {
-    AwsSignatureProcessor signatureProcessor = new AwsSignatureProcessor(mRequestContext);
-    Authenticator authenticator = Authenticator.Factory.create(mSConf);
-    AwsAuthInfo authInfo = signatureProcessor.getAuthInfo();
-    if (authenticator.isAuthenticated(authInfo)) {
-      return authInfo.getAccessID();
-    }
-    throw new S3Exception(authInfo.toString(), S3ErrorCode.INVALID_IDENTIFIER);
-  }
-
-  /**
-   * Gets the user from the authorization header string for AWS Signature Version 4.
-   * @param authorization the authorization header string
-   * @param conf the {@link AlluxioConfiguration} Alluxio conf
-   * @return the user
-   */
-  @VisibleForTesting
-  public static String getUserFromAuthorization(String authorization, AlluxioConfiguration conf)
-      throws S3Exception {
-    if (conf.get(PropertyKey.SECURITY_AUTHENTICATION_TYPE) == AuthType.NOSASL) {
-      return null;
-    }
-    if (authorization == null) {
-      throw new S3Exception("The authorization header that you provided is not valid.",
-              S3ErrorCode.AUTHORIZATION_HEADER_MALFORMED);
-    }
-
-    // Parse the authorization header defined at
-    // https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html
-    // All other authorization types are deprecated or EOL (as of writing)
-    // Example Header value (spaces turned to line breaks):
-    // AWS4-HMAC-SHA256
-    // Credential=AKIAIOSFODNN7EXAMPLE/20130524/us-east-1/s3/aws4_request,
-    // SignedHeaders=host;range;x-amz-date,
-    // Signature=fe5f80f77d5fa3beca038a248ff027d0445342fe2855ddc963176630326f1024
-
-    // We only care about the credential key, so split the header by " " and then take everything
-    // after the "=" and before the first "/"
-    String[] fields = authorization.split(" ");
-    if (fields.length < 2) {
-      throw new S3Exception("The authorization header that you provided is not valid.",
-              S3ErrorCode.AUTHORIZATION_HEADER_MALFORMED);
-    }
-    String credentials = fields[1];
-    String[] creds = credentials.split("=");
-    // only support version 4 signature
-    if (creds.length < 2 || !StringUtils.equals("Credential", creds[0])) {
-      throw new S3Exception("The authorization header that you provided is not valid.",
-          S3ErrorCode.AUTHORIZATION_HEADER_MALFORMED);
-    }
-
-    final String user = creds[1].substring(0, creds[1].indexOf("/")).trim();
-    if (user.isEmpty()) {
-      throw new S3Exception("The authorization header that you provided is not valid.",
-              S3ErrorCode.AUTHORIZATION_HEADER_MALFORMED);
-    }
-
-    return user;
+  private String getUser(String authorization) {
+    return authorization;
```

Comment:
I'd prefer if we just refactor and remove this helper function if you're going to replace the contents of the `"Authorization"` header with the parsed contents.

Reply:
what you mean is we can remove the function `getUser` and directly use the parameter? I can change the name of parameter from `Authorization` to `User`, or We can use another Header Name to present the user name.